modules/ROOT/pages/organization.adoc

```diff
@@ -4,28 +4,24 @@ include::_attributes.adoc[]
 
-When you sign into Anypoint Platform, the organization icon and name are displayed on the top right of the page. You can click the organization icon to navigate between business groups. The master organization can contain multiple business groups. You can think of business groups as sub organizations, or children of the master organization.
+An organization is an account where multiple users can share resources, including applications and environments. The level of access users have to various resources depends on their assigned roles and permissions. For example, one user might have permission to manage API alerts, while another user has permission only to view API alerts.
+
+When you sign into Anypoint Platform, the organization icon and name are displayed on the top right of the page. The master organization can contain multiple business groups. You can think of business groups as sub-organizations, or children of the master organization. Click the organization icon to navigate between business groups. 
 
 [NOTE]
---
-When you sign into Anypoint Platform, you are signed back into whichever organization, or business group, you were in when you last signed out.
---
+When you sign into Anypoint Platform, you are signed back into the organization or business group you were in when you last signed out. Always verify you are in the correct organization or business group.
 
 == Organization Owner
 
-The user who first signs up for an Anypoint Platform account is designated as the organization owner. This is not a role but an identifier for this single user, who is assigned the Organization Administrators role by default. Organization Administrators can view and configure properties for an organization.
-
-Other users can be assigned the Organization Administrators role, but there is only one organization owner per Anypoint Platform account (master organization). The owner can transfer ownership of the account to another user, but there cannot be more than one owner at once.
+The user who first signs up for an Anypoint Platform account is designated as the organization owner. This is not a role that can be assigned, rather it is an identifier for this single user, who is assigned the *Organization Administrators* role by default. 
 
-When the organization owner creates a business group, an owner must be assigned to it. This assigned user holds an Organization Administrators role within that business group by default.
+Other users can be assigned the *Organization Administrators* role, but there is only one organization owner per Anypoint Platform account (master organization). The owner can transfer ownership of the account to another user, but there cannot be more than one owner at once. The *Organization Administrators* role cannot be removed from the organization owner in the master organization or any business group that belongs to the master organization.
```

Comment:
Any org admin can change owners, not just _the_ owner. The new owner must also be an org admin